### ISSUE TITLE
Text input does not work with monthPicker

### ISSUE THREAD

**Issue by weskubo-cgi** [Vuepic/vue3-date-time-picker#89]
In version 2.4.2 of the plugin, text entry of the date does not work when using monthPicker, textInput and format="yyyy-MM". This worked correctly up to version 2.3.6 and appears to be broken in 2.4.0+. It seems that loading of the monthPicker is removing focus from the input.

```
<template>
  <div>
    <Datepicker v-model="date" format="yyyy-MM" monthPicker textInput/>
  </div>
</template>
```

**Comment by Jasenkoo** [comment]
Included in the [v2.5.0](https://github.com/Vuepic/vue3-date-time-picker/releases/tag/v2.5.0)

**Comment by weskubo-cgi** [comment]
Verified in v2.5.0.
